src/SCRIPTS/BF/PAGES/acc_calibrate.lua

```diff
@@ -0,0 +1,30 @@
+
+local template = assert(loadScript(radio.template))()
+local margin = template.margin
+local indent = template.indent
+local lineSpacing = template.lineSpacing
+local tableSpacing = template.tableSpacing
+local sp = template.listSpacing.field
+local yMinLim = radio.yMinLimit
+local x = margin
+local y = yMinLim - lineSpacing
+local inc = { x = function(val) x = x + val return x end, y = function(val) y = y + val return y end }
+local labels = {}
+local fields = {}
+
+if apiVersion >= 1.041 then
+	labels[#labels + 1] = { t = "Trim Accelerometer" ,     x = x,          y = inc.y(lineSpacing) }
+	fields[#fields + 1] = { t = "acc trim pitch",    x = x, y = inc.y(lineSpacing), sp = x + sp, min = -300, max = 300, vals = { 1 } }
+	fields[#fields + 1] = { t = "acc trim roll",     x = x, y = inc.y(lineSpacing), sp = x + sp, min = -300, max = 300, vals = { 2 } }
```

Comment:
Too much indent. 
The field text could also be just "Pitch" and "Roll" since the "Trim Accelerometer" header explains it all. These should also be indented. "x = x + indent" for both of these. Look at the other pages to see how we usually do it.

This won't work as it should. both pitch and roll trim are 16 bit values so for pitch it should be "vals = { 1, 2 }" and for roll "vals = { 3, 4 }". Have a look at the MSP_ACC_TRIM" message in the BF source code. 🙂